Library/Homebrew/utils/inreplace.rb

```diff
@@ -45,7 +43,7 @@ def initialize(errors)
         audit_result: T::Boolean,
       ).void
     }
-    def inreplace(paths, before = nil, after = nil, audit_result = true) # rubocop:disable Style/OptionalBooleanParameter
+    def self.inreplace(paths, before = nil, after = nil, audit_result = true) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
Maybe worth fixing up the `OptionalBooleanParameter` while you're here and fixing all call-sites anyway? I think the best fix would be just making these all kwargs.

Reply:
Sure, done ✅